<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveLogFileDialog</class>
 <widget class="QDialog" name="SaveLogFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Windows</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>210</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Save windows </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="RegsCheckBox">
      <property name="text">
       <string>Registers</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="TextCheckBox">
      <property name="text">
       <string>Text Segments</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="DataCheckBox">
      <property name="text">
       <string>Data Segments</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="ConsoleCheckBox">
      <property name="text">
       <string>Console</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="BrowsePushButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Save to file</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="SaveLineEdit"/>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="SavePushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="AbortPushButton">
      <property name="text">
       <string>Abort</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="TextRadioButton">
      <property name="text">
       <string>Text</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="HTMLRadioButton">
      <property name="text">
       <string>HTML</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SavePushButton</sender>
   <signal>clicked()</signal>
   <receiver>SaveLogFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AbortPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SaveLogFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
